bundle/src/main/java/com/adobe/acs/commons/mcp/util/package-info.java

```diff
@@ -17,7 +17,7 @@
  * limitations under the License.
  * #L%
  */
-@Version("3.11.0")
+@Version("4.0.0")
```

Comment:
see other comments on how this can be potentially avoided.

Reply:
I set the provider type annotations but it still is requiring the major bump (unfortunately) -- any other ideas?